reference/configuration/tidb-server/configuration.md

```diff
@@ -80,6 +80,11 @@ category: reference
 + 默认："/tmp/tidb"
 + 可以通过 `tidb-server --store=mocktikv --path=""` 来启动一个纯内存引擎的 TiDB
 
+## `--tmp-storage-path`
+
++ TiDB 临时磁盘存储位置。
++ 默认：`<操作系统临时文件夹>/tidb/tmp-storage`
```

Comment:
we should also mention that this config item takes effect only when `oom-use-tmp-storage` is set to `true`

Reply:
It's already stayed here. See https://github.com/pingcap/docs-cn/pull/2306/files#diff-03d37f540e23cadd32b8e1c4c20d4b76R33

Reply:
I'll remove this config in another PR.